### PR TITLE
Added a Hacktoberfest themed banner for repo

### PR DESCRIPTION
**Pull request by chiragkhatri19**
#### Describe your changes
Added a Hacktoberfest themed banner for repo

#### Link to issue this resolves
https://github.com/brittanyjoiner15/eui-event-template/issues/86

#### Screenshot of changes(if relevant)
Screenshot 2022-10-07 102340

* [x] The **title** should be something descriptive about what you're changes do. _(It will default to whatever you put as your commit message.)_
* [x] This **description** should include ...
  * [x] A short summary of your changes
  * [x] A link to the issue that relates to these changes
  * [x] A screenshot of the changes if possible!
* [x] Add `brittanyjoiner15` as a reviewer
* [x] Add yourself as assignee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![Frame 1Britany banner](https://user-images.githubusercontent.com/89431059/194469940-b753d9d3-7d63-4b22-b144-76afbf228a92.png)
```
Comment (brittanyjoiner15) [inline README.md]: can we make the text something like `Hacktoberfest themed banner with repo name`

**Review by brittanyjoiner15 — CHANGES_REQUESTED** [review]
can you update the alt text with my recommendation? and then we'll be good to merge! this looks great!!

**Comment by chiragkhatri19** [comment]
Yeah you can just tell me what text content you want. I'll edit it real quick

**Comment by brittanyjoiner15** [comment]
@chiragkhatri19 can you make it `Hacktoberfest themed banner for repo`

**Review by brittanyjoiner15 — APPROVED** [review]
Looks great! Thanks for your work!

**Pull request merged**